app-website/src/layouts/GameLayout.astro

```diff
@@ -0,0 +1,11 @@
+---
+import BaseLayout from "./BaseLayout.astro";
+const { frontmatter } = Astro.props;
```

Comment:
Does frontmatter refer metadata? Either accept a complete game model or individual props (seriously, the former though). Perhaps just a naming issue?